tests/integration/local/start_api/test_start_api.py

```diff
@@ -148,6 +148,39 @@ def test_large_input_request_http10(self):
         self.assertEqual(response.raw.version, 11)
 
 
+class TestParallelRequestsZipped(StartApiIntegBaseClass):
+    template_path = "/testdata/start_api/template.yaml"
+
+    def setUp(self):
+        self.url = "http://127.0.0.1:{}".format(self.port)
+        HTTPConnection._http_vsn_str = "HTTP/1.1"
+
+    @pytest.mark.flaky(reruns=3)
+    @pytest.mark.timeout(timeout=600, method="thread")
+    def test_same_endpoint_zipped(self):
+        """
+        Send two requests to the same path at the same time. This is to ensure we can handle
+        multiple requests at once and do not block/queue up requests
+        """
+        number_of_requests = 10
```

Comment:
The comment says two request, but the variable is set to 10, is it suppose to be 10 requests?

Reply:
Yeah, Ill change the comment